source/config/configSpec.py

```diff
@@ -47,6 +47,7 @@
 # Audio settings
 [audio]
 	audioDuckingMode = integer(default=0)
+	idleCloseDelay = integer(default=10000,min=0,max=86400000)

```

Comment:
Wow, a day is a bit long isn't it? May be leave out the max, or set it somewhere around an hour.